lib/mayaUsd/ufe/ProxyShapeHierarchy.cpp

```diff
@@ -248,6 +249,13 @@ ProxyShapeHierarchy::reorderCmd(const Ufe::SceneItemList& orderedList) const
     return UsdUndoReorderCommand::create(getUsdRootPrim(), orderedTokens);
 }
 
+Ufe::UndoableCommand::Ptr ProxyShapeHierarchy::ungroupCmd() const
```

Comment:
I think this should return a nullptr.  You can't ungroup the pseudo-root: it doesn't have a parent.